### ISSUE TITLE
Trying to run on macOS

### ISSUE THREAD

**Issue by claudioluciano** [grow-graphics/gd#15]
Hey, i'm trying to run a example on macOS but i only get a blank screen and on the editor a bunch on broken nodes.

`ERROR: Can't open dynamic library: /Users/../Development/pong/graphics/library.framework. Error: dlopen(/Applications/Godot.app/Contents/MacOS/../Frameworks/library.framework, 0x0002): tried: '/Applications/Godot.app/Contents/MacOS/../Frameworks/library.framework' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Applications/Godot.app/Contents/MacOS/../Frameworks/library.framework' (no such file), '/Applications/Godot.app/Contents/MacOS/../Frameworks/library.framework' (no such file).`

The readme does not properly explain how to make things work on mac, i have a look on the `gd` cli and found the cli looks for an specific binary on the PATH, and i think works fine on a linux or windows idk, but the mac version of godot the CLI is bundle with the APP it's self, to workaround that, i have create a symlink with the name that `gd` was looking for and put that on the PATH.

Godot v4.2.2.stable.official

go version go1.22.2 darwin/arm64

**Comment by Splizard** [comment]
Thanks for reporting! I'll see if I can improve the experience on Mac.

**Comment by claudioluciano** [comment]
> Should be fixed, please reinstall/upgrade the `gd` command and try again. It will use `godot`binary if it is the correct version, otherwise if `brew` is available, it will try to install it that way. The Godot library configuration should also be correct now.


It's working!!!

one thing that it's not working is the godot install using brew.
```
❯ brew install godot@4.2.2
Warning: No available formula with the name "godot@4.2.2".
==> Searching for similarly named formulae and casks...
==> Casks
godot@3

To install godot@3, run:
  brew install --cask godot@3
```

installing godot with `brew install godot` without specify the version, works fine.
anyways, there plans to make the debugger support?
